adhocracy-plus/assets/scss/icons/style.css

```diff
@@ -81,6 +82,10 @@
     content: "\e90a";
 }
 
+.icon__img {
```

Comment:
instead of using css I think we could just change the thumbnail tag from `0x40` to `0x32`